statics/js/components/inject-form.js

```diff
@@ -22,10 +22,13 @@ Vue.component('inject-form', {
                        id="inject-src"\
                        attr="id"\
                        v-model="node1"></node-selector>\
+        <input id="src-ip" v-if="noSrcIP" class="form-control input-sm inject-target" v-model="srcIP" placeholder="No IP in src node, please enter"/>\
```

Comment:
I think it could be nice add a way to force the IP as for most of the interface with the IPV6 stack have a link-local address.

Reply:
so we need to get the IP always from user? if no IP given by user should we have to throw a form error?

Reply:
not always but having a way (button or something else) to allows the user to force the IP.

Reply:
got your point. thanks!

Reply:
I think it's better to show ip in the box when this node has ip addr
This will  looks more clearly